from_cpython/Lib/test/test_long.py

```diff
@@ -17,7 +16,10 @@ def __str__(self):
         return self.format % self.args
 
 # SHIFT should match the value in longintrepr.h for best testing.
-SHIFT = sys.long_info.bits_per_digit
+SHIFT = 30
```

Comment:
Did you pick 30 since it's cpython's value?  I would argue for 64 here to match GMP's internal structure (though it probably doesn't matter that much since I doubt we'll expose bugs in GMP) and since it seems like a higher value mostly means more testing.